### ISSUE TITLE
LoTr - Courtyard and DRC-related issues

### ISSUE THREAD

**Issue by belohoub** [SmartCAMPUSZCU/KETCube-mainBoard#10]
In commit 85befbb27f99d5ea245dd573626dabc8a712ef8b , I did several changes and I included library changes fixing the courtyard issues with DRC - please could you try to remove as much DRC warnings related to coutryard as possible? 

I revisited many parts in our library and it seems, that majority of them had correct courtyard, only those with incorrect one were changed. 

Courtyard could be intentionally ignored if necessary for this board (yes, it is too small), however, it should not if possible ... there are several parts, whose could further be moved to solve the courtyard issue.

**Comment by RuzickaOndra** [comment]
I agree, three of them remained because  I think there is no better option. All the others are resolved.
